docker/scripts/build-sepolia-docker.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+sudo docker build -t snapchain/sepolia-cdk-validium-contracts -f docker/Dockerfile.sepolia .
```

Comment:
so this need to change `snapchain/sepolia-cdk-validium-contracts` to `snapchain/cdk-validium-contracts`